NewForce-Cohort-9/newforce-c9-tabloidmvc-TabloidMVC
# Added register function

I have created the register function for the app so that new users can register their profile. This coincides with the #29 trello ticket https://trello.com/c/dsizqWLn/29-29-register

STEPS TO TEST
1. After making sure you have our repo cloned down into a local repository and have the database added, git fetch --all in the master branch to retrieve the branch named rsbranch that I have pushed up.
2. Git checkout rsbranch to switch to my branch and open up the project (TabloidMVC) in Visual Studio.
3. Then, run the application in a browser window. You should see the 'Home' page with a navigation bar at the top of the page. If not already logged out, log out (with the link in the navbar).
4. Then, click the reigster button under the login button.
5.  When promted to register a new user, please enter a name in the name tab, a last name in the last name tab, an email in the email tab, and an email in the email tab. Once all the values are filled in, click submit. 
6. After you submit the values, In visual studio run the database in the SQL Object explorer and check to see if the values you submitted for the register function are recorded in the database.

Checklist 
* My code follows the style guidelines of this project
* I have performed a self-review of my code
* I have commented my code, particularly in hard-to-understand areas
* My changes generate no new warnings or errors
* I have added test instructions that prove my fix is effective or that my feature works
